docs/en/latest/deployment-modes.md

```diff
@@ -293,7 +295,7 @@ routes:
 #END
 ```
 
-*WARNING*: APISIX will not load the rules into memory from file `conf/apisix.yaml` if there is no `#END` at the end.
+*WARNING*: APISIX will not load the rules into memory from file `conf/apisix.yaml` if there is no `#END` at the end. However, when using `conf/apisix.json`, the `#END` marker is not required, as APISIX can directly parse and validate the JSON structure.
```

Comment:
You need to clarify the priority of the `apisix.yaml` and `apisix.json` configuration files.
When both files exist, which one is valid.